content/applications/hr/payroll/payslips.rst

```diff
@@ -28,159 +30,235 @@ from this dashboard.
-  app.
-- :guilabel:`Period`: Click the default date to reveal a pop-up calendar. On this calendar, use the
-  :guilabel:`< (less-than)` and :guilabel:`> (greater than)` icons to select the desired month, and
-  click on the desired day to select that specific date as the start date for the payslip. Repeat
-  this process to add an end date for the payslip in the field below.
-- :guilabel:`Contract`: Using the drop-down menu, select the desired contract for the employee. Only
-  the available corresponding contracts for the selected employee appear as options.
-- :guilabel:`Structure`: Using the drop-down menu, select the salary structure type. Only the
-  available corresponding structures for the selected contract for that specific employee appear as
-  options.
-- :guilabel:`Payslip Name`: In the blank field, type in the name for the payslip. The name should be
-  short and descriptive, such as `April 2023`.
-- :guilabel:`Company`: In the :guilabel:`Accounting Information` tab, select the company the payslip
-  applies to from the :guilabel:`Company` drop-down menu.
-- :guilabel:`Salary Journal`: In the :guilabel:`Accounting Information` tab, enter the salary
-  journal in which the payment will be reflected, and found in the *Accounting* application.
-
-.. image:: payslips/new-payslip-tab.png
-   :align: center
-   :alt: The necessary fields for a new payslip in the Accounting Information tab.
+   :alt: The top fields for a new payslip all filled out for a February payslip.
+
+.. note::
+   Typically, after making a selection in the :guilabel:`Employee` field, Odoo auto-populates all
+   other required fields (besides the :guilabel:`Period` field), but **only** if that information is
+   already on that employee's form in the *Employees* app.
 
 .. important::
-   It is recommended to check with the accounting department to ensure every entry that affects the
-   *Accounting* application is correct.
-
-Optional fields
-~~~~~~~~~~~~~~~
-
-- :guilabel:`Reference`: Any note or reference message for the new entry can be entered here.
-- :guilabel:`Company Car`: If applicable, select the company car from the drop-down.
-- :guilabel:`Worked Days`: In the :guilabel:`Worked Days \& Inputs` tab, the entries under
-  :guilabel:`Worked Days` (including the :guilabel:`Type`, :guilabel:`Description`,
-  :guilabel:`Number of Days`, :guilabel:`Number of Hours`, and :guilabel:`Amount`) are automatically
-  filled in, based on what was entered for the :guilabel:`Period`, :guilabel:`Contract`, and
-  :guilabel:`Structure` fields of the payslip form.
-- :guilabel:`Salary Computation`: The :guilabel:`Salary Computation` tab is automatically filled in
-  after the :guilabel:`Compute Sheet` button is clicked. Doing so displays all the wages,
+   If modifications to auto-populated fields are made, it is recommended to check with the
+   accounting department to ensure every entry that affects the *Accounting* application is correct.
+
+Worked days & inputs tab
```

Comment:
I would typically agree, but with the new style standards, we were supposed to be using this format. I have changed all the headings to be in this case, so since I feel it is more in alignment with the guidelines, I am going to keep this as it is.